sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppImpl.java

```diff
@@ -358,4 +379,37 @@ public SpringAppImpl withoutConfigurationServiceBinding() {
         configurationServiceConfigs.put(Constants.BINDING_RESOURCE_ID, "");
         return this;
     }
+
+    @Override
+    public SpringAppImpl withServiceRegistryBinding() {
+        ensureProperty();
+        Map<String, Map<String, Object>> addonConfigs = innerModel().properties().addonConfigs();
+        if (addonConfigs == null) {
+            addonConfigs = new HashMap<>();
+            innerModel().properties().withAddonConfigs(addonConfigs);
+        }
+        SpringServiceRegistry serviceRegistry = parent().getDefaultServiceRegistry();
```

Comment:
it there non-default service registry?

Reply:
Not from portal or our premium code. There's exposed API from swagger to do so, but I think it can be ignored.